react-container/Dockerfile

```diff
@@ -14,9 +16,12 @@ RUN npm install
  
 # Copies everything over to Docker environment
 COPY . .
+
+# Builds prod image
+RUN npm run build
  
 # Uses port which is used by the actual application
-EXPOSE 3000
+EXPOSE 5000
 
 # Finally runs the application
-CMD [ "npm", "start" ]
\ No newline at end of file
+CMD [ "serve", "-s", "build" ]
```

Comment:
@JerHowden I changed this because npm start is running a development version of the app that requires more overhead. When you run "npm build" which transpiles the React files to a production ready package, it tells you to use "serve"